drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/client/DriftNettyMethodInvokerFactory.java

```diff
@@ -117,7 +116,7 @@ public void close()
         }
         finally {
             try {
-                group.shutdownGracefully(0, 0, MILLISECONDS).await();
+                group.shutdownGracefully().await();
```

Comment:
curious. what's the difference here? 😃 . Since the old says there is 0 millisecond quite period and 0 millisecond timeout...